experimental/LieAlgebras/src/AbstractRootSystem.jl

```diff
@@ -0,0 +1,93 @@
+###############################################################################
+#
+#   Root System
+#
+###############################################################################
+
+struct AbstractRootSystem <: RootSystem
+  # familiy (A, ..., G) of the root system
+  fam::Symbol
```

Comment:
General root systems are supported, this was used as a helper for printing. For now I have removed this. In general we can only print the Cartan matrix defining the root system, since detecting the Dynkin type of the Cartan matrix is not easy.
I am not sure what a nice `show` for a root system (and Weyl group) would be; feel free to make suggestions.